app/scripts/metamask-controller.js

```diff
@@ -483,15 +483,12 @@ export default class MetamaskController extends EventEmitter {
       trackMetaMetricsEvent: (...args) =>
         this.metaMetricsController.trackEvent(...args),
     });
-    this.networkController.initializeProvider();
+    this.networkController.initializeProvider(false);
     this.provider =
       this.networkController.getProviderAndBlockTracker().provider;
     this.blockTracker =
       this.networkController.getProviderAndBlockTracker().blockTracker;
 
-    // TODO: Delete when ready to remove `networkVersion` from provider object
-    this.deprecatedNetworkId = null;
```

Comment:
Hmm, why was this moved? We reference this when initializing a dapp connection, which we can expect to happen during onboarding. This would effectively mean that this gets initialized as `undefined` now instead of `null`, then gets changed to `null` later. before being updated for the first time. That doesn't sound useful.

Reply:
addressed in `f8b430639d909e506efed60a60d26b7030f16ac3`